pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -267,6 +267,18 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 			volumeToAttach.VolumeSpec, volumeToAttach.NodeName)
 
 		if attachErr != nil {
+			if derr, ok := attachErr.(*util.DanglingAttachError); ok {
+				addErr := actualStateOfWorld.MarkVolumeAsAttached(
```

Comment:
Can we `glog.Info` (or warning) when we're doing this ... it's unusual enough that we want to know in the logs I think